playground/mocks/data/idp/idx/error-authenticator-verification-passcode-change-on-prem.json

```diff
@@ -7,9 +7,12 @@
     "type": "array",
     "value": [
       {
-        "message": "Wait for token to change, then enter the new tokencode.",
+        "message": "Pin accepted, Wait for token to change, then enter new passcode",
```

Comment:
Is the string really going to include a capitalized word after a comma? Should be a period instead of a comma?

Reply:
This is the example i got from the backend. These error messages will be from the RSA server which we have no control over. The backend will just pass the error message as-is to the UI.